students.txt

```diff
@@ -13,6 +13,9 @@ List of students
 6. Vladimir S.
 7. Daniil I.
 8. Daria P. 
-9. Max R.
+9. Max Rakovskyi
+Love videogames, especially Dota2. I like playing basketball and travelling to the mountains.
```

Comment:
`to the mountains` -> `to mountains`
And please, start each sentence from a new line.